src/main/java/Car.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Random;
+
+public class Car {
+    private final String name;
+    private int position = 0;
+    private Random random = new Random();
+    public Car(String name){
+        this.name = name;
+    }
+    public void run(){
+        position += goForward(random.nextInt(10));
+    }
+    private int goForward(int randnum){
+        if(randnum<=3)
```

Comment:
블럭내 코드가 한 줄일 경우 brace {}를 생략할 수 있죠! 그러나 brace를 생략하실 때는 구문의 옆에 작성해주세요!

```suggestion
        if(randnum <= 3) return 0;
```